modules/layers/src/column-layer/cylinder-geometry.js

```diff
@@ -0,0 +1,14 @@
+import TruncatedConeGeometry from './stroke-truncated-cone-geometry';
+import {uid} from '@luma.gl/core';
+
+export default class CylinderGeometry extends TruncatedConeGeometry {
+  constructor(props = {}) {
```

Comment:
The subclassing seems excessive. We can just have one custom geometry that is always a cylinder.